modules/33acrossBidAdapter.js

```diff
@@ -1,5 +1,6 @@
 const { registerBidder } = require('../src/adapters/bidderFactory');
-const utils = require('../src/utils');
+const { userSync } = require('../src/userSync');
```

Comment:
Do import over here.  `import { userSync } from 'src/userSync';` 
This will make sure it imports same object https://github.com/prebid/Prebid.js/blob/master/src/userSync.js#L189